apps/testing-javascript/src/data/add-srt-to-specific-video.ts

```diff
@@ -0,0 +1,50 @@
+// Running this script:
+//
+// Be sure to set `MUX_ACCESS_TOKEN` and `MUX_SECRET_KEY` in your
+// `.env.local` environment variables file. And assign string values
+// to the `assetId` and `srtUrl` variables below.
+//
+// Execute the script with:
+//
+// ```
+// npx ts-node --files --skipProject src/data/add-srt-to-specific-video.ts
+// ```
+
+import Mux from '@mux/mux-node'
+
+require('dotenv-flow').config({
+  default_node_env: 'development',
+})
+
+const assetId = undefined // Set me!
+const srtUrl = undefined // Set me!
+
+const addSrtToSpecificVideo = async (
```

Comment:
personally I found it useful to check here if it already exists and remove it first